MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+- During the community part of the Review Phase: There will be a period of time for the community to carefully read and consider the MIP and the Review Group’s Risk Assessment. The MIP Facilitator will coordinate with the Mina Foundation to organize a community call where the MIP Author and the MIP Review Group can explain the MIP, including its risks, benefits and tradeoffs, in a way that is comprehensible to non-expert audiences and respond to community questions. Dates for the on-chain vote will be set, along with instructions on how to participate.
+
+### Changes in the Drafting and Review Phases
+
+**Drafting Phase - MIP Submission**
+
+During the Drafting phase, when the MIP Author submits a new MIP as a pull request in GitHub, a MIP Facilitator is assigned to assist the MIP Author, review the MIP and (if necessary, consult with the MIP Reviewers to) decide whether to approve or reject the pull request according to the following acceptance criteria:
+- The proposal is original or new.
+- The proposal is clearly articulated.
+- The proposal uses the MIP Template correctly.
+- The proposal is not missing any required parts.
+
+If the pull request is rejected, the MIP Facilitator will communicate the decision to the MIP Author explaining the reasons for the rejection and a list of changes that could be made for the proposal to be accepted, so that the MIP Author can make these changes.
+
+If the pull request is accepted, the MIP Facilitator:
+- Approves the MIP and assigns it a formal MIP number.
+- Changes the status of the MIP.
+- Merges in the PR.
+- Communicates the decision to the MIP author.
+
+**Review Phase - Part 1: Risk Assessment**
+We propose to create a pool of experts to review MIPs across the 3 categories. For each MIP, the Author reaches out to the pool of experts for the relevant category and sends a Request for Comments. To maintain independence, the Author does not choose who the Reviewers are. This MIP Review Group has a period to review the MIP and provide their feedback. The MIP Author can incorporate the feedback into the MIP until a final draft is completed. If the MIP Author does not incorporate the feedback, the MIP process still continues.
+
+When the MIP Author considers the draft ready for the final review, editing stops for at least 1 week to allow MIP Reviewers to write individually a Risk Assessment that is incorporated into the designated section of the MIP template. 
+
+The MIP Facilitator must check that every MIP has enough reviews to complete the Review phase (at least 3 if there is agreement, 5 otherwise). The MIP Facilitator can also coordinate communications and collaborate with the MIP Author to schedule any necessary meetings, and to define a roadmap with target dates for each milestone in the MIP process. 
```

Comment:
How do we ensure there are enough reviewers in each category to reach these thresholds of 3 or 5 reviews?